lib/media_player.py

```diff
@@ -28,7 +28,13 @@ def is_playing(self):
 class VLCMediaPlayer(MediaPlayer):
     def __init__(self):
         MediaPlayer.__init__(self)
-        self.instance = vlc.Instance()
+
+        options = (
+          '--video-x=1920 '
```

Comment:
Not sure we want to hardcode this. It might be 4k. Or even a lower resolution. Can we read this dynamically?

Reply:
I tried removing the `--video-x` option. As long as `--mmal-display=hdmi-2` is present, it works. Still figuring out if mirror mode is possible.